apps/wallet/src/ui/app/pages/home/tokens/OverviewHint.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { type ElementType } from 'react';
+
+interface OverviewHintProps {
+    onClick: () => void;
+    icon: ElementType;
+    title: string;
+}
+
+export function OverviewHint({ onClick, icon, title }: OverviewHintProps) {
+    const IconComponent = icon;
+    return (
+        <div
+            className="state-layer relative flex w-full cursor-pointer items-center gap-3 rounded-xl  border border-transparent bg-neutral-96 p-xs px-sm py-xs dark:bg-neutral-10"
+            onClick={onClick}
+        >
+            <IconComponent className="h-5 w-5 text-primary-30 dark:text-primary-80" />
+            <span className="text-label-sm text-neutral-10 dark:text-neutral-92">{title}</span>
```

Comment:
This way it matches the style of "Start Staking"
```suggestion
            <IconComponent className="h-5 w-5 text-primary-20 dark:text-primary-90" />
            <span className="text-label-sm text-neutral-10 dark:text-neutral-92">{title}</span>
```

Reply:
it's not what's in figma, but I agree with you! I'll update it, thanks! :saluting_face: